Automattic-Tracks-iOS/UI/Crash Logging/CrashLoggingView.swift

```diff
@@ -0,0 +1,97 @@
+import SwiftUI
+import Sentry
+
+@available(iOS 13.0, OSX 10.15, *)
+public struct CrashLoggingView: View {
+
+    @State
+    var sendErrorAndWaitStatus: SendErrorAndWaitStatus = .none
+
+    @State
+    var sendingError: Error?
+
+    private let crashLogging: CrashLogging
+
+    public init(crashLogging: CrashLogging) {
+        self.crashLogging = crashLogging
+    }
+
+    public var body: some View {
+        Form {
+            Section(header: Text("Actions")) {
+                Button("Send Test Crash", action: sendTestCrash)
+                Button("Send Test Event", action: sendTestEvent)
+                HStack {
+                    Button(action: sendErrorAndWait) {
+                        HStack {
+                            Text("Send Error and Wait")
+
+                            /// Nicely align the icon to the right on iOS but don't leave
+                            /// a weird-shaped button on macOS
+                            #if os(iOS)
+                            Spacer()
+                            #endif
+
+                            switch sendErrorAndWaitStatus {
+                                case .none:
+                                    Group {} /// An empty view
+                                case .uploading:
+                                    Text("⏳")
+                                case .success:
+                                    Text("✅")
+                                case .error:
+                                    Text("⚠️")
+                            }
```

Comment:
I'm a big fan of using enums to describe mutually exclusive states in the UI. Nicely done 😄